sample/SipInterconnect/public/stylesheets/pattern.css

```diff
@@ -0,0 +1,773 @@
+@import url("https://fast.fonts.com/t/1.css?apiType=css&projectid=eab16f51-e7e4-4285-8b2b-3844b7921c78");
```

Comment:
i dont like the idea of committing the pattern library CSS to this repo, it should be imported from a link tag and hosted on a CDN. but since we don't have that set up right now, its cool.